bedrock/careers/templates/careers/includes/testimonials.html

```diff
@@ -5,50 +5,52 @@
 #}
 {% block careers_content %}
   <section class="c-careers-testimonials c-careers-section no-js-hidden">
-    <div class="mzp-l-content">
-      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two c-careers-testimonials-wrapper">
-        <li class="has-modal" data-modal-id="perla-d">
-          <p>
-            "Despite a diverse set of backgrounds, we share a similar passion and need for data and browser transparency..."
-          </p>
+      <ul class="mzp-l-content c-careers-testimonials-wrapper">
+        <li class="c-careers-testimonial-item has-modal" data-modal-id="perla-d">
+          <q>
+            Despite a diverse set of backgrounds, we share a similar passion...
+          </q>
           <a href="#perla-d-testimonial">
-          Read full testimonial
+          Read full Quote
```

Comment:
**Nit:** no need for `Quote` to be capitalized, let's change it to lowercase throughout the section:
```suggestion
          Read full quote
```

Reply:
Sounds good, just grabbed what was in the design